ts/model_service_worker.py

```diff
@@ -193,7 +193,8 @@ def run_server(self):
 
     except socket.timeout:
         logging.error("Backend worker did not receive connection in: %d", SOCKET_ACCEPT_TIMEOUT)
-    except Exception:  # pylint: disable=broad-except
+    except Exception as e:  # pylint: disable=broad-except
+        logging.error(e)
```

Comment:
original line 197 already sets exc_info=True, so it is not necessary to add this.

Reply:
Hi @lxning,
Could you please explain a bit more? Do you mean calling `logger.error(e)` is unnecessary?

To explain a bit more, sometime I can see the error message from `logging.error("Backend worker process died.", exc_info=True)`, but sometimes can't not.
It seems that after the worker is restarted (`Worker disconnected. WORKER_STARTED`), error message is truncated somehow.
So normally when debugging, I edit this file to add `logger.error(e)` to read error message easier.